desktop/cmp/persistenceManager/PersistenceManager.ts

```diff
@@ -0,0 +1,214 @@
+import {div, fragment, hbox} from '@xh/hoist/cmp/layout';
+import {hoistCmp, HoistProps, PlainObject, uses} from '@xh/hoist/core';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {Icon} from '@xh/hoist/icon/Icon';
+import {menu, menuDivider, menuItem, popover} from '@xh/hoist/kit/blueprint';
+import {groupBy, keys, sortBy} from 'lodash';
+import {ReactNode} from 'react';
+import {manageDialog} from './impl/ManageDialog';
+import {saveDialog} from './impl/SaveDialog';
+import './PersistenceManager.scss';
+import {PersistenceManagerModel} from './PersistenceManagerModel';
+
+interface PersistenceManagerModelProps extends HoistProps {
+    model?: PersistenceManagerModel;
+    /** True to disable options for saving/managing items. */
+    minimal?: boolean;
+}
+
+export const [PersistenceManager, persistenceManager] =
+    hoistCmp.withFactory<PersistenceManagerModelProps>({
+        displayName: 'PersistenceManager',
+        model: uses(PersistenceManagerModel),
+
+        render({model, minimal = false}) {
+            const {
+                selectedObject,
+                isShared,
+                capitalPluralNoun,
+                manageDialogModel,
+                saveDialogModel
+            } = model;
+
+            return fragment(
+                hbox({
+                    className: 'persistence-manager',
+                    items: [
+                        popover({
+                            item: button({
+                                text: selectedObject?.name
```

Comment:
Nit - since `getHierarchyDisplayName` is null-safe, could simplify this slightly with `text: getHierarchyDisplayName(selectedObject?.name) ?? capitalPluralNoun`